### PR TITLE
isConnected field was added

### PR DESCRIPTION
**Pull request by IamEgor**
Opportunity to get connection status

### CODE REVIEW

**Comment by IamEgor** [comment]
@Gurgen Could you please add isConnected method for WebSocketHubConnection

File: dotnetcoresignalrclient/src/main/java/com/smartarmenia/dotnetcoresignalrclient/WebSocketHubConnection.kt [hunk 1]
```diff
@@ -27,6 +27,9 @@ class WebSocketHubConnection(private val hubUrl: String) : HubConnection {
     private val eventListeners = mutableMapOf<String, MutableList<HubEventListener>>()
     private val parsedUri = Uri.parse(hubUrl)
     private val gson = Gson()
+    @Volatile
+    var isConnected = false
```
Comment (Gurgen) [inline dotnetcoresignalrclient/src/main/java/com/smartarmenia/dotnetcoresignalrclient/WebSocketHubConnection.kt]: I think we don't need this boolean.

**Comment by IamEgor** [comment]
Should I do changes or revert them for Kotlin classes?

**Comment by Gurgen** [comment]
No need make any change to kotlin project, we are using java project. I will delete Kotlin version

File: dotnetcoresignalrclient/src/main/java/com/smartarmenia/dotnetcoresignalrclient/WebSocketHubConnection.kt [hunk 2]
```diff
@@ -81,6 +85,7 @@ class WebSocketHubConnection(private val hubUrl: String) : HubConnection {
 
             override fun onClose(i: Int, s: String, b: Boolean) {
                 Log.i(TAG, "Closed. Code: $i, Reason: $s, Remote: $b")
+                isConnected = false
```
Comment (Gurgen) [inline dotnetcoresignalrclient/src/main/java/com/smartarmenia/dotnetcoresignalrclient/WebSocketHubConnection.kt]: Remove

**Comment by Gurgen** [comment]
Thank you @IamEgor

**Pull request merged**